bin/get-localized-urls

```diff
@@ -36,7 +36,9 @@ https.get('https://resources.scratch.mit.edu/localized-urls.json', (res) => {
     res.on('end', () => {
         try {
             var urlJson = JSON.parse(urlData);
-            fs.writeFile(filename, JSON.stringify(urlJson, null, '  '));
+            fs.writeFile(filename, JSON.stringify(urlJson, null, '  '), (err) => {
```

Comment:
Now that you are handling the error properly you can drop the try/catch.

Reply:
@thisandagain I think the original intention of the `try/catch` was to handle any JSON parsing errors in the line above (and writeFile errors just weren't being seen). I think I should move the `writeFile` line out of the `try/catch`, but still keep it around

Reply:
Then I think for the sake of being terse, I would recommend you keep the existing functionality and simply use `fs.writeFileSync`.